.github/workflows/audit.yml

```diff
@@ -37,4 +37,4 @@ jobs:
 
     - name: Check Secrets SDK Vulnerabilities
       working-directory: packages/secrets/src/keyring
-      run: cargo audit --deny warnings --ignore RUSTSEC-2024-0370
+      run: cargo audit --deny warnings --ignore RUSTSEC-2024-0370 --ignore RUSTSEC-2024-0429
```

Comment:
[RUSTSEC-2024-0429](https://rustsec.org/advisories/RUSTSEC-2024-0429.html) should be safe to ignore since there is no CVE associated with it and we don't use the affected function.

If it was easy to update the `glib` crate I would do so, but we're kind of locked down to a specific version for maximum compatibility with `libsecret` across multiple distros.